src/visualization/include/iDynTree/MeshcatVisualizer.h

```diff
@@ -58,16 +58,21 @@ class MeshcatVisualizer
                        const iDynTree::Span<const double> &jointPositions,
                        const std::string &modelName);
 
+    bool set_sphere(const std::string &name,
+                    const iDynTree::Position &position,
+                    const double radius,
+                    const iDynTree::VectorDynSize &color);
```

Comment:
In https://github.com/robotology/idyntree/blob/c453ba094a707ab6399eeda740922bdc9cf94a84/src/model/include/iDynTree/SolidShapes.h#L24 we represent `Color` as a `Vector4` , probably we can use it also here? (Or perhaps for consistency we can use `iDynTree::Span<const double>`? cc @GiulioRomualdi

Reply:
The color here should be rgb only since it is used within the following method without alpha channel:  
void MeshcatCpp::Material::set_color(uint8_t r, uint8_t g, uint8_t b);

Reply:
In the end `iDynTree::Span<const double>` has been used. It needs to be a vector of 4 elements between 0 and 1. The alpha channel has been enabled